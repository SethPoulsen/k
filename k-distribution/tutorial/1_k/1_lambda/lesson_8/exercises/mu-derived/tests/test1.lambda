
letrec sumTo x = if x <= 1 then 1 else (x + (sumTo (x + -1)))
in (sumTo 1000)