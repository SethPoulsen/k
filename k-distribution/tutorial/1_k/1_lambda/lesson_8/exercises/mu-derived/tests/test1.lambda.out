<k>
  500500
</k>
